apps/dashboard/app/views/files/_favorites.html.erb

```diff
@@ -5,7 +5,12 @@
     <ul id="favorites" class="nav nav-pills flex-column">
       <li role="presentation" class="nav-item"><%= link_to 'Home Directory', files_path(Dir.home), class: "nav-link d bg-light" %></li>
       <% OodFilesApp.new.favorite_paths.each do |p| %>
-        <li class="nav-item"><%= link_to p.title || p.path.to_s, files_path(p.filesystem, p.path.to_s), class: "nav-link d bg-light" %>
+        <li class="nav-item">
+          <%= link_to files_path(p.filesystem, p.path.to_s), class: "nav-link d bg-light" do %>
+          <%= fa_icon "cloud", classes: '' %>

```

Comment:
I've got to look this over - it looks like you've hard coded `cloud` icons for all favorite paths, so I'm either misreading it or something.

Reply:
Hmm, let me double check that. Seems like some code may have gone missing.

Reply:
It has indeed gone missing. Added it now.